app/views/posts/index.html.erb

```diff
@@ -9,5 +9,21 @@
 
   <body>
     <h1>Here is a list of posts</h1>
+    <div class="root-container posts-comments-container">
+      <%= render 'shared/user_info', user: @user %>
+      <div class="posts-comments">
+        <% @posts.each_with_index do |post, post_no| %>
+          <div class="post-comments">
+            <%= render 'shared/post_info', post: post, post_no: post_no %>
+            <div class="comments">
+              <% post.comments.each do |comment| %>
```

Comment:
Please use `#five_recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb page`.